src/pivot-table/components/__tests__/shared-styles.test.ts

```diff
@@ -4,11 +4,35 @@ import { borderStyle, cellStyle, getBorderStyle } from "../shared-styles";
 describe("Shared styles", () => {
   describe("getBorderStyle", () => {
     const borderColor = "red";
-    const baseStyle = { ...cellStyle, ...borderStyle, borderColor, borderWidth: 0 };
+    const baseStyle = {
+      ...cellStyle,
+      ...borderStyle,
+      borderWidth: 0,
+      borderRightColor: undefined,
+      borderBottomColor: undefined,
+    };
     const noBorder = { ...baseStyle, borderRightWidth: 0, borderBottomWidth: 0 };
-    const borderRight = { ...baseStyle, borderRightWidth: 1, borderBottomWidth: 0 };
-    const borderBottom = { ...baseStyle, borderRightWidth: 0, borderBottomWidth: 1 };
-    const borderRightBottom = { ...baseStyle, borderRightWidth: 1, borderBottomWidth: 1 };
+    const borderRight = {
+      ...baseStyle,
+      borderRightWidth: 1,
+      borderBottomWidth: 0,
+      borderRightColor: borderColor,
+      borderBottomColor: undefined,
```

Comment:
This is already undefined in baseStyle, so no need to override unless for readability